windows/wix.json

```
@@ -32,6 +32,19 @@
   "directories": [
     {
       "name": "plugins"
+    },
+    {
+      "name": "storage"
+    }
+  ],
+  "environments": [
+    {
+      "name": "OIQ_OTEL_COLLECTOR_STORAGE",
+      "value": "C:\\Program Files\\observIQ OpenTelemetry Collector",
+      "permanent": "yes",
```

```suggestion
      "permanent": "no",
```

We should remove this env variable on uninstall